kod/object/passive/spell/radiusench/bane.kod

```diff
@@ -0,0 +1,116 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+BaneEnchantment is RadiusEnchantment
+
+constants:
+
+   include blakston.khd
+
+resources:
+   
+   default_Hurt_Caster = "A tide of energy retaliates against your spell."
+   default_Kills_You = "The oncoming tide of energy carries you straight to "
+                       "the Underworld!"
+
+classvars:
+   
+   Bane_Hurt_Caster = default_Hurt_Caster
+   Bane_Kills_You = default_Kills_You
```

Comment:
These should be of the format vrBaneHurt or vrBaneKilled in line with existing classvars (v) holding resources (r).